belgif-rest-problem/src/main/resources/io/github/belgif/rest/problem/Messages_fr.properties

```diff
@@ -0,0 +1,30 @@
+# problem detail messages
+BadGatewayProblem.detail=Probl�me de communication avec le service en amont
+BadRequestProblem.detail=Le message en input est incorrect
+ExpiredAccessTokenProblem.detail=Le jeton d'acc�s Bearer trouv� dans le Header HTTP Authorization est expir�
+InvalidAccessTokenProblem.detail=Le jeton d'acc�s Bearer trouv� dans le Header HTTP Authorization est invalide
+NoAccessTokenProblem.detail=Aucun jeton d'acc�s Bearer trouv� dans le Header HTTP Authorization
+ResourceNotFoundProblem.detail=Aucune donn�e trouv�e pour %s avec %s [%s]
+# issue detail messages
+unknownInput.detail=L'input %s est inconnu
+outOfRange.minmax.detail=La valeur de l'input %s = %s est en dehors de l'intervalle [%s, %s]
+outOfRange.min.detail=La valeur de l'input %s = %s doit au moins �tre �gale � %s
+outOfRange.max.detail=La valeur de l'input %s = %s ne peut pas �tre sup�rieure � %s
+referencedResourceNotFound.detail=La ressource r�f�renc�e %s = '%s' est inconnue
+referencedResourceNotFound.ssin.detail=Le NISS %s est inconnu
+rejectedInput.detail=L'input %s n'est pas autoris� dans ce contexte
+requiredInput.detail=L'input %s est obligatoire dans ce contexte
+requiredInput.ifPresent.detail=Tous ces inputs doivent �tre pr�sents lorsque %s est pr�sent: %s
+replacedSsin.detail=Le NISS %s a �t� remplac� par %s
+canceledSsin.detail=Le NISS %s a �t� annul�
+invalidStructure.ssin.detail=Le NISS %s est invalide
+invalidStructure.incompleteDate.detail=La date incompl�te %s est invalide
+invalidStructure.yearMonth.detail=L'ann�e et le mois %s sont invalides
+invalidStructure.enterpriseNumber.detail=Le num�ro d'entreprise %s est invalide
+invalidStructure.establishmentUnitNumber.detail=Le num�ro de l'unit� d'�tablissement %s est invalide
+invalidPeriod.detail=%s ne peut pas �tre ant�rieur � %s
+exactlyOneOfExpected.detail=Exactement un de ces inputs doit �tre pr�sent: %s
+anyOfExpected.detail=Au moins un de ces inputs doit �tre pr�sent: %s
+zeroOrExactlyOneOfExpected.detail=Exactement un de ces inputs ou aucun input ne doit �tre pr�sent: %s
```

Comment:
@jflabatBCSS Doesn't the "ne doit" also apply to "Exactement un de ces inputs"?

Reply:
I'm not sure, the "ne doit" is for "aucun input" but I can't use for "Exactement un de ces inputs". Maybe "Exactement un de ces inputs ou aucun input doit être présent" or "Exactement un de ces inputs doit être présent ou aucun input ne doit être présent" are better. I'll ask @usku01 for her opinion.

Reply:
I changed the sentence